azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/DeletedSecretBundle.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+package com.microsoft.azure.keyvault.models;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A Deleted Secret consisting of its previous id, attributes and its tags, as
+ * well as information on when it will be purged.
+ */
+public class DeletedSecretBundle extends SecretBundle {
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.